src/features/user/RegisterTrees/RegisterTreesWidget.tsx

```diff
@@ -341,14 +353,14 @@ function RegisterTreesForm({
             )}
           </div>
           <div className={`${styles.locationMap}`}>
-            {isMultiple ? (
+            {isMultiple && isStyleReady ? (
               <DrawMap setGeometry={setGeometry} userLocation={userLocation} />
             ) : (
               <MapGL
```

Comment:
There is a typescript error here : ` Type 'number | number[]' is not assignable to type 'number | undefined'.`. You could resolve this by checking if the value is array or not using Array.isArray()

Reply:
The error was related to the 'zoom' type, which was supposed to be a number but was incorrectly defined as 'number[].